etc/magesetup.xml

```diff
@@ -22,6 +22,8 @@
             <tax__calculation__shipping_includes_tax>1</tax__calculation__shipping_includes_tax>
             <tax__calculation__apply_after_discount>1</tax__calculation__apply_after_discount>
             <tax__calculation__discount_tax>1</tax__calculation__discount_tax>
+            <tax__calculation__cross_border_trade_enabled>0</tax__calculation__cross_border_trade_enabled>
```

Comment:
Why has this been added? Doesn't this mean we change this setting during our setup? I think we should not touch this during our setup, because it is not required for any market (especially not for the German market).

Reply:
well, I don't have strong opinion here - can remove that as well.

Reply:
Please remove this for the reasons I mentioned above.